Add OCDs for Italy

zikowang opened the pull request:
Adding OCDs for Italy's senate and camera.

100 for Senate
- 26 Collegio plurinominale
- 74 Collegio uninominale

196 for Camera
- 49 Collegio plurinominale
- 147 Collegio uninominale

HKSenior commented on identifiers/country-it.csv:
```diff
@@ -1,2 +1,346 @@
+ocd-division/country:it/senato:campania/cu:5,Collegio uninominale Campania - 05
+ocd-division/country:it/senato:campania/cu:6,Collegio uninominale Campania - 06
+ocd-division/country:it/senato:campania/cu:7,Collegio uninominale Campania - 07
+ocd-division/country:it/senato:diaspora-_africa_asia_oceania_e_antartide,Abroad - Africa Asia Oceania and Antarctica
```
The dash can be removed here and below.

```suggestion
ocd-division/country:it/senato:diaspora_africa_asia_oceania_e_antartide,Abroad - Africa Asia Oceania and Antarctica
```

zikowang replied:
adjusted

HKSenior commented on identifiers/country-it/senato.csv:
```diff
@@ -0,0 +1,125 @@
+ocd-division/country:it/senato:veneto,Circoscrizione Veneto
+ocd-division/country:it/senato:diaspora-_europa,Abroad - Europe
+ocd-division/country:it/senato:diaspora-_america_settentrionale_e_centrale,Abroad - North and Central America
+ocd-division/country:it/senato:diaspora-_america_meridionale,Abroad - South America
```
The dash can be removed here and below.

```suggestion
ocd-division/country:it/senato:diaspora_america_meridionale,Abroad - South America
```

zikowang replied:
adjusted

zikowang commented on identifiers/country-it/README.md:
```diff
@@ -0,0 +1,16 @@
+  - [senato (senate / upper house)](https://dait.interno.gov.it/territorio-e-autonomie-locali/sut/elenco_collegi_senato.php)
+  - Collegio plurinominale as **cp**
+  - Collegio uninominale as **cu**
+* Collegio plurinominale and Collegio uninominale of the Camera of Italy
```
Good call, added english translation to readme.

zikowang commented:
@HKSenior thanks a lot for the review.

The changes requested changes have been implemented.

jpmckinney commented:
Instead of `cameri` and `senato`, how about `lower` and `upper`? That way jurisdictions can be comparable, instead of having local type IDs for lower and upper chambers of parliament.

The readme links to regions, but regions are not the same as Circoscrizione. There can be multiple Circoscrizione per region, for the lower house: https://it.wikipedia.org/wiki/Collegi_elettorali_della_Camera_dei_deputati_del_2020

It also looks like the single-member constituencies should be children of the multi-member constituencies.

Overall, I think the hierarchy would be:

* region
  * upper (circoscrizione) – can maybe skip this level if we assume that the relationship will remain 1-to-1 in future
    * mmc (multi-member constituency)
      * smc or ed (single-member constituency or electoral district)
  * lower (circoscrizione)
    * mmc (multi-member constituency)
      * smc or ed (single-member constituency or electoral district)

jpmckinney commented:
Looks good!

> There are no multi-member constituencies for the Circoscrizione Valle D'Aosta and Trentino-Alto Adige in [senate](https://dait.interno.gov.it/territorio-e-autonomie-locali/sut/elenco_collegi_senato.php).
And there are no multi-member constituencies for the Circoscrizione Valle D'Aosta in [camrea](https://dait.interno.gov.it/territorio-e-autonomie-locali/sut/elenco_collegi_camera.php).

Aha, for those, I guess `ed` will be a child of `upper`/`lower`.

zikowang commented:
Alright, I updated the files, adde Regions and adjusted the OCDs for upper and lower house.

I also enhanced the name of the single-member constituencies.

Please have another look.

zikowang commented:
> This touches country-lt.csv (Lithuania) as well?

Good catch! That was mistakenly pushed -> fixed.

sguenther85 commented:
> * This touches country-lt.csv (Lithuania) as well?
> * Although I had suggested it, now that I see it, I think the `region:` level is repetitive, and also might imply that the mmc and ed are with respect to the regional government, whereas these are about representation at the federal level. Therefore, it might be better to remove the region level. Curious to hear from other maintainers.

Hi, I would also suggest leaving out the region. They can be problematic for several reasons:
- the same thing you said @jpmckinney : might imply that the mmc and ed are with respect to the regional government. That's why the ed in germany are directly under country for the “federal election”
- if constituencies with numbers/names could span or switch regions
- generally no added value, except more work

zikowang commented:
@jpmckinney should I remove the `region:` level so we can move on?

jpmckinney commented:
Yes please!

zikowang commented:
Alright. Regions removed from country IT OCDs.

Please have a look again.
